src/status_im2/subs/wallet/wallet_test.cljs

```diff
@@ -84,9 +86,12 @@
   [sub-name]
   (testing "a map: address->balance"
     (swap! rf-db/app-db #(assoc-in % [:wallet :accounts] accounts))
+    (let [result      (rf/sub [sub-name])
+          balance-0x1 (money/bignumber 3250)
+          balance-0x2 (money/bignumber 2100)]
 
-    (is (= {"0x1" 3250 "0x2" 2100}
-           (rf/sub [sub-name])))))
+      (is (money/equal-to balance-0x1 (get result "0x1")))
+      (is (money/equal-to balance-0x2 (get result "0x2"))))))
```

Comment:
This file is now testing against bignumber instances. the function `utils.money/equal-to` is needed to check equality between them.